docs/source/changelog.md

```diff
@@ -1,8 +1,56 @@
-# Summary of changes in Jupyter Kernel Gateway
+# Summary of changes
 
-See `git log` for a more detailed summary.
+See `git log` and `CHANGELOG.md` for a more detailed summary of changes.
```

Comment:
Are we going to have to copy/paste the CHANGELOG into the docs now before every release? Would be nice if the CHANGELOG automatically got pulled into the sphinx build. (Little bit of make magic?)